### PR TITLE
FF114 WebTransport API - Release note/Exper. features

### PR DESCRIPTION
**Pull request by hamishwillee**
FF114 ships WebTransport API support, including all interfaces (though not necessarily all parts of all interfaces).

This removes the experimental features note, and adds a release note.

There are some flaws because corresponding docs have not yet merged.

Other docs work can be tracked in #26684

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/114/index.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ This article provides information about the changes in Firefox 114 that affect d
 
 - [`Window.print()`](/en-US/docs/Web/API/Window/print) now opens a print dialog on Firefox for Android, allowing the current document to be printed ([Firefox bug 1809922](https://bugzil.la/1809922)).
 
+- The [WebTransport API](/en-US/docs/Web/API/WebTransport_API) has been implemented, including all interfaces: [`WebTransport`](/en-US/docs/Web/API/WebTransport), [`WebTransportBidirectionalStream`](/en-US/docs/Web/API/WebTransportBidirectionalStream) , [`WebTransportDatagramDuplexStream`](/en-US/docs/Web/API/WebTransportDatagramDuplexStream) ,[`WebTransportReceiveStream`](/en-US/docs/Web/API/WebTransportReceiveStream), [`WebTransportDatagramDuplexStream`](/en-US/docs/Web/API/WebTransportDatagramDuplexStream) and [`WebTransportError`](/en-US/docs/Web/API/WebTransportError).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/114/index.md:47]: Wording suggestion and some comma / whitespace fixes:

```suggestion
- The [WebTransport API](/en-US/docs/Web/API/WebTransport_API) is now supported, which includes the following interfaces: [`WebTransport`](/en-US/docs/Web/API/WebTransport), [`WebTransportBidirectionalStream`](/en-US/docs/Web/API/WebTransportBidirectionalStream), [`WebTransportDatagramDuplexStream`](/en-US/docs/Web/API/WebTransportDatagramDuplexStream), [`WebTransportReceiveStream`](/en-US/docs/Web/API/WebTransportReceiveStream), [`WebTransportDatagramDuplexStream`](/en-US/docs/Web/API/WebTransportDatagramDuplexStream) and [`WebTransportError`](/en-US/docs/Web/API/WebTransportError).
```

**Review by bsmth — APPROVED** [review]
There's a few suggestions, but leaving a +1 in advance. Thanks Hamish 👍🏻

File: files/en-us/mozilla/firefox/releases/114/index.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ This article provides information about the changes in Firefox 114 that affect d
...
+  For more information see [Firefox bug 1692754](https://bugzil.la/16927541), [Firefox bug 1818754](https://bugzil.la/1818754), and [Firefox bug 1791835](https://bugzil.la/1791835).
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/114/index.md:48]: Thanks. Actually 181332 was deliberately omitted, but poor cleanup.

**Comment by hamishwillee** [comment]
Thanks very much for the review @bsmth

**Pull request merged**